.github/CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+/components/ide @gitpod-io/engineering-ide
+/components/ide-proxy @gitpod-io/engineering-ide
+/components/image-builder-api @csweichel @geropl
+/components/image-builder-bob @gitpod-io/engineering-workspace
+/components/image-builder-mk3 @gitpod-io/engineering-workspace
+/components/installation-telemetry @gitpod-io/engineering-self-hosted
+/components/licensor @gitpod-io/engineering-webapp
+/components/local-app-api @csweichel @akosyakov
+/components/local-app @gitpod-io/engineering-ide
+/components/openvsx-proxy @gitpod-io/engineering-ide
+/components/proxy @gitpod-io/engineering-webapp
+/components/registry-facade-api @csweichel @aledbf
+/components/registry-facade @gitpod-io/engineering-workspace
+/components/server @gitpod-io/engineering-webapp
+/components/service-waiter @gitpod-io/engineering-webapp
+/components/supervisor-api @csweichel @akosyakov
+/components/supervisor @gitpod-io/engineering-ide
+/components/workspacekit @gitpod-io/engineering-workspace
+/components/ws-daemon-api @csweichel @aledbf
+/components/ws-daemon @gitpod-io/engineering-workspace
+/components/ws-manager-api @csweichel @aledbf
+/components/ws-manager-bridge-api @csweichel @geropl
+/components/ws-manager-bridge @gitpod-io/engineering-webapp
+/components/ws-manager @gitpod-io/engineering-workspace
+/components/ws-proxy @gitpod-io/engineering-workspace
+/dev/gpctl @gitpod-io/engineering-workspace
+/dev/loadgen @gitpod-io/engineering-workspace
+/installer @gitpod-io/engineering-self-hosted
+/installer/pkg/components/agent-smith @gitpod-io/engineering-workspace
+/installer/pkg/components/blobserve @gitpod-io/engineering-workspace
+/installer/pkg/components/components-webapp @gitpod-io/engineering-webapp
+/installer/pkg/components/components-workspace @gitpod-io/engineering-workspace
+/installer/pkg/components/content-service @gitpod-io/engineering-workspace
+/installer/pkg/components/dashboard @gitpod-io/engineering-webapp
+/installer/pkg/components/ide-proxy @gitpod-io/engineering-ide
+/installer/pkg/components/image-builder-mk3 @gitpod-io/engineering-workspace
+/installer/pkg/components/openvsx-proxy @gitpod-io/engineering-ide
+/installer/pkg/components/proxy @gitpod-io/engineering-webapp
+/installer/pkg/components/registry-facade @gitpod-io/engineering-workspace
+/installer/pkg/components/server @gitpod-io/engineering-webapp
+/installer/pkg/components/server/ide @gitpod-io/engineering-ide
+/installer/pkg/components/workspace @gitpod-io/engineering-workspace
+/installer/pkg/components/workspace/ide @gitpod-io/engineering-ide
+/installer/pkg/components/ws-daemon @gitpod-io/engineering-workspace
+/installer/pkg/components/ws-manager-bridge @gitpod-io/engineering-webapp
+/installer/pkg/components/ws-manager @gitpod-io/engineering-workspace
+/installer/pkg/components/ws-proxy @gitpod-io/engineering-workspace
+/installer/pkg/config/versions @gitpod-io/engineering-ide @gitpod-io/engineering-ide @gitpod-io/engineering-ide @gitpod-io/engineering-ide
```

Comment:
I think once is enough

Reply:
haha this turned out to be a bug. Fixing in upcoming PR

Reply:
Was fixed in https://github.com/gitpod-io/gitpod/pull/8000/files and there were not other cases of multiple teams being approvers in the same OWNERS file